website/pages/en/billing.mdx

```diff
@@ -2,15 +2,38 @@
 title: Billing
 ---
 
-## Billing on Arbitrum
-
 > Invoices are generated on a weekly basis.
 
+There are two options for paying for your query fees:
+
+- [Paying with fiat currency with Banxa](#billing-on-banxa)
+- [Paying with crypto wallet](#billing-on-arbitrum)
+
+## Billing on Banxa
+
+Banxa enables you to bypass the need for an exchange and pay for your query fees using the fiat currency of your choice. The fiat currency will be converted to GRT, added to your account balance on the billing contract, and used to pay for queries associated with your API keys.
+
+There may be KYC requirements depending on the regulations in your country. For more information about KYC, please visit [Banxa's FAQ page](https://docs.banxa.com/docs/faqs).
+
+You can learn more about Banxa by reading their [documentation](https://docs.banxa.com/docs).
+
+### Paying for query fees with Banxa
+
+1. Select “Pay with Card” option in [Subgraph Studio](https://thegraph.com/studio/billing/?show=Deposit).
+2. Click the 'Continue with Banxa' button.
+3. Enter the amount of GRT to be added to your account balance.
```

Comment:
2. Enter the amount of GRT to be added to your account balance.
3. Click the 'Continue with Banxa' button.

The order of these two lines should be switched since 'Continue with Banxa' is grayed out until an amount is included. That number is then carried forward to Banxa.

Reply:
Got that cleaned up! Great catch, thank you!